controllers/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -272,7 +272,13 @@ func (r *PublishingStrategyReconciler) Reconcile(ctx context.Context, request ct
 
 	// If the OCM API sends an empty applicationIngress array, and we are on a version greater than 4.13, assume that
 	// we want to 'disown' the native ingress controller. Any remaining ingresses will be deleted as per usual.
+	// We also ensure that the scope of the default API server ingress matches the scope of the publishing strategy CR.
 	if baseutils.IsVersionHigherThan("4.13") && !ownedIngressExistingMap["default"] {
+		result, err = r.ensureAliasScope(reqLogger, instance, clusterBaseDomain)
```

Comment:
We should move `ensureAliasScope` out as we want the scope of default API server to match scope of publishing strategy CR for both native ingress controller and managed ingress